src/controllers/meteorsController.js

```diff
@@ -1,17 +1,17 @@
-const express = require('express');
-const router = express.Router();
 const meteorsService = require('../services/meteorsData');
-const MeteorRequestDto = require('../dto/meteorRequestDto');
+const MeteorsRequestDto = require('../dto/meteorRequestDto');
 
-router.get('/', async function (req, res, next) {
-  const { startDate, endDate, countOnly, wereDangerous } = new MeteorRequestDto(req.query);
+const getMeteors = async (req, res, next) => {
+  const { startDate, endDate, countOnly, wereDangerous } = new MeteorsRequestDto(req.query);
 
   try {
     res.json(await meteorsService.getMeteorsData(startDate, endDate, countOnly, wereDangerous));
   } catch (err) {
     console.error('Error while getting meteors data ', err.message);
```

Comment:
probably, you can move `console.error()` to errorHandler
This will help you store all processing in one place

Reply:
Fixed